packages/core/src/render3/component_ref.ts

```diff
@@ -527,118 +526,10 @@ export class ComponentRef<T> extends AbstractComponentRef<T> {
   }
 }
 
+// TODO: remove?
```

Comment:
nit: expanding the content of a comment a bit to provide more context (please check if my comment is actually correct :)):
```suggestion
// TODO: this type seems to be unused, should we remove it?
```